.github/workflows/tests.yml

```diff
@@ -0,0 +1,105 @@
+name: 'Tests'
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.number }}-${{ github.event.ref }}
+  cancel-in-progress: true
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
+
+permissions:
+  contents: read
+
+jobs:
+  pytest:
+    name: '${{ matrix.os }} / ${{ matrix.python }} / ${{ matrix.distrib }} / ${{ matrix.sphinx_version }}'
+    timeout-minutes: 30
+    continue-on-error: true
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        shell: bash -el {0}
+    env:
+      PYTHON_VERSION: '${{ matrix.python }}'
+      SPHINX_VERSION: '${{ matrix.sphinx_version }}'
+      DISTRIB: '${{ matrix.distrib }}'
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            python: '3.12'
+            sphinx_version: dev
+            distrib: mamba
+            locale: C
+          - os: ubuntu-latest
+            python: '3.11'
+            sphinx_version: '6'
+            distrib: mamba
+          - os: ubuntu-latest
+            python: '3.8'
```

Comment:
I assume 3.8 is the min python version we support? Not essential, but is it easy to make this a variable? Makes it easier to update and maybe easier to find/more explicit what our min support python version is.

Reply:
> but is it easy to make this a variable?

You mean in YAML? Not super easy. I think it's not too bad to find it here. I'll just update `developers.rst` to mention this alongside "latest Python" in the release instructions/reminders.